Tests/PointFreeTests/HomeTests.swift

```diff
@@ -11,19 +11,86 @@ import WebKit
 #endif
 
 class HomeTests: TestCase {
-  func testHomepage() {
+  override func setUp() {
+    super.setUp()
+
+    let eps = [
+      ep10,
+      ep2,
+      ep1,
+      introduction,
+      ]
+      .suffix(4)
+      .map(\.image .~ "")
```

Comment:
so easy to clear out all the images on an episode so that we dont do URL requests.